homework-1/js/script.js

```diff
@@ -0,0 +1,136 @@
+////////////menu///////////
+const iconMenu = document.querySelector('.menu__icon');
+  if(iconMenu) {
+    const menuBody = document.querySelector('.menu__body');
+    iconMenu.addEventListener("click", function(e){
```

Comment:
You have alredy used one single quotes. Try to keep one approach

Reply:
fixed a bug

Reply:
done